marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -140,6 +140,10 @@ class BrowsePage {
     return cy.get('#selected-facets [data-cy="clear-' + facet + '"]');
   }
 
+  getAppliedFacetName(facet: string) {
```

Comment:
Can we make use of data-testid when its available. I want to ultimately get rid of all data-cy, since data-testid can work both in e2e and RTL tests.
<img width="993" alt="image" src="https://user-images.githubusercontent.com/16124406/90814662-4bc46280-e2de-11ea-9c2f-8c859d8bc996.png">

And since this is a button, can you try if .contains("text") work.

Reply:
Changed to data-testid locator. We shouldn't use contains() for this step because the exact string needs to be verified. e.g. expected "email: coleadams39@nutralab.com" not "Customer.email: coleadams39@nutralab.com"